### DISCUSSION TITLE
Dead link in wiki

### DISCUSSION THREAD

**Discussion by p-bakker** [eclipse-egit/egit#44] [Q&A]
Looks like https://github.com/eclipse-egit/egit/wiki/Contributor-Guide#java-requirements links to non-existing wiki page: https://github.com/eclipse-egit/egit/wiki/Execution_Environments#Installing_Execution_Environment_Descriptions

Was in the process of creating an issue, but dunno in which category this would fall. Doesn't feel like a bug...

**Comment by tomaswolf** [comment]
This appears to have been fixed in the meantime, the link now goes to https://github.com/eclipse-pde/eclipse.pde/blob/master/docs/Execution_Environments.md .
